x-pack/packages/kbn-elastic-assistant/impl/assistant_context/types.tsx

```diff
@@ -16,6 +16,7 @@ export interface MessagePresentation {
 export interface Message {
   role: ConversationRole;
   reader?: ReadableStreamDefaultReader<Uint8Array>;
+  replacements?: Record<string, string>;
```

Comment:
note: Not written to individual `Message` in localstorage, but rather the entire conversation as intended